src/Images.jsx

```diff
@@ -178,15 +179,19 @@ class Images extends React.Component {
                 {_("Get new image")}
             </Button>
         ];
+
+        const openedInterm = this.state.intermediateOpened;
```

Comment:
Let's please be consistent in the alias names, or avoid the alias at all.

Reply:
Changed